API-strategie-modules/access-control/access-control.md

```diff
@@ -155,15 +156,25 @@ Using tokens, a distinction is made between authorized and non-authorized servic
 
 In case the proper headers are not sent, then there are no authentication details available and a status error code `403 Forbidden` is returned.
 
-<div class="rule" id="api-52">
-  <p class="rulelab"><strong>API-52</strong>: Use OAuth 2.0 for authorization with rights delegation</p>
+**How to test**
+Testing is conditional on a correct Open API Specification(OAS) being present as well as an access token being provided by the API provider to the test client. If this is the case an automated test client can test if the access control method works according to this rule and the method specified in the OAS spec is indeed implemented. Negative tests are out of scope as there are to many variables to account for when testing other (not allowed) methods for providing tokens.
+
+<span id="api-52"></span>
+<div class="rule" id="/access-control/OAuth">
+  <p class="rulelab"><strong>="/access-control/OAuth</strong>: Use OAuth 2.0 for authorization with rights delegation</p>
   <p>This is in line with the way the OAuth standard appears on the comply or explain list of Forum Standaardisatie.</p>
 </div>
 
+**How to test**
+Any test is conditional upon the API provider informing the test client in some way that rights delegation is applicable. IF this is the case the test client can identify if the endpoints required by NL GOV Assurance profile for OAuth 2.0 are present and active. A full compliance test of this profile is out of scope for this module.
```

Comment:
Vraag, als `type: oauth2` in de OAS staat dan moet het voldoen aan NL GOV Assurance profile for OAuth 2.0? Of is dat optioneel?

Zo ja, dan zou voor het testen handig zijn dat het profiel actief gebruikt moet worden. Ook al valt testen nu buiten scope.